JASP-Engine/JASP/R/MixedModelsCommon.R

```diff
@@ -644,6 +651,95 @@
     
     return()
   }
+.mmFitStats      <- function(jaspResults, options, type = "LMM") {
+  if (!is.null(jaspResults[["fitStats"]]))
+    return()
+  
+  model <- jaspResults[["mmModel"]]$object$model
+  
+  fitStats <- createJaspTable(title = gettext("Fit Statistics"))
+  fitStats$position <- 2
+
+  if (type == "LMM") {
+    dependencies <- .mmDependenciesLMM
+  } else if (type == "GLMM") {
+    dependencies <- .mmDependenciesGLMM
+  }
+  if (options$method == "PB") {
+    dependencies <- c(dependencies, "seed", "setSeed")
+  }
+
+
+  fitStats$dependOn(c(dependencies, "fitStats"))
+
+  
+  if (is.list(model$full_model)) {
+    is_REML <-
+      lme4::isREML(model$full_model[[length(model$full_model)]])
+  } else{
+    is_REML <- lme4::isREML(model$full_model)
+  }
+  
+  fitStats$addColumnInfo(name = "deviance",
+                         title = gettext("deviance"),
```

Comment:
```suggestion
                         title = gettext("Deviance"),
```

Reply:
Maybe? I don't have a strong opinion on this but I have a feeling we tend to have capitals...

Reply:
Yeah, we actually use "Deviance" in other analyses, so change that please